src/System.Windows.Forms/src/System/Windows/Forms/ComboBox.ComboBoxChildEditUiaProvider.cs

```diff
@@ -128,6 +123,27 @@ patternId switch
             ///  Gets the runtime ID.
             /// </summary>
             internal override int[] RuntimeId => new int[] { RuntimeIDFirstItem, GetHashCode() };
+
+            internal override UiaCore.ITextRangeProvider DocumentRangeInternal => _textProvider.DocumentRange;
+
+            internal override UiaCore.ITextRangeProvider[]? GetTextSelection() => _textProvider.GetSelection();
+
+            internal override UiaCore.ITextRangeProvider[]? GetTextVisibleRanges() => _textProvider.GetVisibleRanges();
+
+            internal override UiaCore.ITextRangeProvider? GetTextRangeFromChild(UiaCore.IRawElementProviderSimple childElement)
+                => _textProvider.RangeFromChild(childElement);
```

Comment:
[consistency]
```suggestion
            internal override UiaCore.ITextRangeProvider DocumentRangeInternal
                => _textProvider.DocumentRange;

            internal override UiaCore.ITextRangeProvider[]? GetTextSelection()
                => _textProvider.GetSelection();

            internal override UiaCore.ITextRangeProvider[]? GetTextVisibleRanges()
                => _textProvider.GetVisibleRanges();

            internal override UiaCore.ITextRangeProvider? GetTextRangeFromChild(UiaCore.IRawElementProviderSimple childElement)
                => _textProvider.RangeFromChild(childElement);
```


Ditto in the other files

Reply:
I'm not sure, that it's right. We decided, that we won't move a code to the next line if it's short (less than 120 characters). We use this approach for the rest types (consistency), eg. `AccessibleObject.cs`. These methods are "short" (~110 characters), but they are almost "long".
Ok, I will move these methods to be consistent for one pattern interface here. But what do you think about the rest "short" methods/properties, should we move them to the new line and how long should they be (how many characters)?

Reply:
You're asking hard questions :) I don't know.

Reply:
I would propose to keep our current approach - 120 characters, as Tanya proposed before. And move ~110 characters lines if it would look better in each specific case. Don't you mind?

Reply:
Sounds good